containers/Containerfile

```diff
@@ -0,0 +1,76 @@
+FROM registry.fedoraproject.org/fedora-toolbox:34
+
+MAINTAINER "Candlepin Team" <candlepin@redhat.com>
+
+ENV SMDEV_CONTAINER_OFF='True'
+
+RUN dnf install --disablerepo=updates -y \
+    cmake \
+    dbus-daemon \
+    dnf-utils \
+    gcc \
+    gettext \
+    git \
+    intltool \
+    json-c-devel \
+    langpacks-as \
+    langpacks-bn \
+    langpacks-de \
+    langpacks-es \
+    langpacks-fr \
+    langpacks-gu \
+    langpacks-hi \
+    langpacks-it \
+    langpacks-ja \
+    langpacks-kn \
+    langpacks-ko \
+    langpacks-ml \
+    langpacks-mr \
+    langpacks-or \
+    langpacks-pa \
+    langpacks-pt_BR \
+    langpacks-ru \
+    langpacks-ta \
+    langpacks-te \
+    langpacks-zh_CN \
+    langpacks-zh_TW \
+    libdnf-devel \
+    libnotify-devel \
+    make \
+    npm \
+    openssl \
+    openssl-devel \
+    python3-coverage \
+    python3-dateutil \
+    python3-decorator \
+    python3-devel \
+    python3-ethtool \
+    python3-gobject-base \
+    python3-iniparse \
+    python3-inotify \
+    python3-nose \
+    python3-pip \
+    python3-polib \
+    python3-rpm \
+    python3-simplejson \
+    python3-six \
+    python3-virtualenv \
+    redhat-rpm-config \
+    rpm-build \
+    rpmlint \
+    subscription-manager-rhsm-certificates \
+    sudo \
+    vim \
+    wget \
+    && dnf clean all
+
+RUN mkdir /build
+COPY requirements.txt dev-requirements.txt /build/
```

Comment:
So we are still duplicating the requirements? COPY should be relative, won't
```
COPY ../requirements.txt ../dev-requirements.txt /build/
```
work?

Reply:
So I have a different proposal, we shall see if the tests like it.
I've tried the above and it seems podman is not a fan of relative references that are outside the build context.
I also tried making the local references symlinks to the files in the root of the project and podman also does not like following symlinks apparently.

Current solution is to put the actual files in the containers directory and make symlinks at the top level.

Reply:
Ok